v2.0/enable-node-map.md

```diff
@@ -4,4 +4,199 @@ summary: Learn how to enable the node map in the Admin UI.
+
+To get the latitudes and longitudes of common AWS/Azure/GC regions, see [Locations Coordinates for Reference](#location-coordinates-for-reference).
+
+### Step 5. View the Node Map
+
+[Open the **Overview page**](admin-ui-overview.html) and select **Node Map** from the **View** drop-down menu to view the **Node Map**.
+
+### Step 6. Navigate the Node Map
+
+Let's say you want to navigate to Node 2, which is in datacenter `us-east-1a` in the `us-east-1` region. To navigate to Node 2:
+
+1. Click on map component marked as **region=us-east-1** on the **Node Map**. The datacenter view is displayed.
+2. Click on the datacenter component marked as **datacenter=us-east-1a**. The individual node components are displayed.
+3. To navigate back to the cluster view, either click on **Cluster** in the bread-crumb trail at the top of the **Node Map**, or click **Up to region=us-east-1** and then click **Up to Cluster** in the lower left-hand side of the **Node Map**.
+
+## Troubleshoot the Node Map
+
+### Node Map Not Displayed
+
+The **Node Map** won't be displayed until all nodes have localities and are assigned the corresponding latitudes and longitudes. To verify if you have assigned localities as well as latitude and longitudes assigned to all nodes, navigate to the Localities debug page (`https://<address of any node>:8080/#/reports/localities`) in the Admin UI.
+
+The Localities debug page displays the following:
+
+- Localities configuration that you set up while starting the nodes with the `--locality` flags.
+- Nodes corresponding to each locality.
+- Latitude and longitude coordinates for each locality/node.
+
+On the page, ensure that every node has a locality as well as latitude/longitude coordinates assigned to them.
+
+### Unable to Assign Latitude/Longitude Coordinates to Localities
+
+You won't be able to assign latitude/longitude coordinates to localities is if the components of your localities have the same name. For example, consider the following partial configuration:
+
+|  Node | Region | Datacenter |
+|  ------ | ------ | ------ |
+|  Node1 | us-east | datacenter-1 |
+|  Node2 | us-west | datacenter-1 |
+
+In this case, if you try to set the latitude/longitude coordinates to the datacenter level of the localities, you will get the "primary key exists" error and the **Node Map** won't be displayed. You can, however, set the latitude/longitude coordinates to the region components of the localities, and the **Node Map** will be displayed.
+
+### Node Map Not Displayed for All Locality Levels
+
+The **Node Map** is displayed only for the locality levels that have latitude/longitude coordinates assigned to them: 
+
+- If you assign the latitude/longitude coordinates at the region level, the **Node Map** shows the regions on the world map. However, when you drill down to the datacenter and further to the individual nodes level, the world map disappears and the datacenters/nodes are be plotted in a circular layout.  
```

Comment:
Remove `level `in `individual nodes level`.

Reply:
Remove `be` in `are be plotted`.

Reply:
Done.